compiler/plugins/input/Torch/PluginRegistration.cpp

```diff
@@ -57,8 +57,12 @@ struct TorchSession
       OpPassManager &passManager, std::string_view typeMnemonic) override {
     if (typeMnemonic == "onnx") {
       // ONNX input is a pre-processing step to torch.
-      passManager.addNestedPass<func::FuncOp>(
-          mlir::torch::onnx_c::createTorchOnnxToTorchPass());
+      mlir::torch::Torch::TorchLoweringPipelineOptions torchOnnxPipelineOptions;
+      torchOnnxPipelineOptions.backendLegalOps = {"aten.flatten.using_ints",
+                                                  "aten.unflatten.int"};
```

Comment:
This could use a comment. What is special about these two ops? Are there `aten.flatten.using_floats` and `aten.unflatten.float` that _aren't_ legal, when the `int` versions are?

Reply:
It's that stableHLO decomposes these ops into `aten.view`, which we inevitably need to convert to a reshape instead of an expand or collapse op.

Reply:
I will add a comment for this.